spec.md

```diff
@@ -3237,14 +3237,14 @@ There are some important things for the SP and CO to consider here:
 
 - It is possible that the gRPC stream gets interrupted for arbitrary reasons beyond
   the control of either the SP or the CO.
-  The SP will get an error when writing to the stream and must abort its transmission.
+  The SP will get an error when writing to the stream and MUST abort its transmission.
   The CO will receive an error in its language specific idiom, which
   will be different from the language specific idiom for the end of a gRPC stream.
 
 - In all circumstances where the CO receives an error when reading from the gRPC stream
-  it may attempt to **continue** the operation by re-sending its request message but with a
-  starting_offset adjusted to a byte position beyond that of any metadata already received.
-  The SP must always examine the starting_offset and ensure that metadata sent in
+  it MAY attempt to **continue** the operation by re-sending its request message but with a
```

Comment:
nit: one sentence per line please (there are several sentences in this section that violate this rule)

Reply:
Fixing!